### PR TITLE
Update cookbook document

### PR DESCRIPTION
**Pull request by daisy20170101**
1) Update cookbook links to SeisSol repository. 2) Fix some installation instructions of using Supermuc NG

### CODE REVIEW

File: Documentation/tpv29.rst [hunk 1]
```diff
@@ -31,10 +31,10 @@ the model.
    Line to the end: fault topography of each nodes (nx\*ny, in meters)
    
 
-Save this file as *mytopo\_tpv29*.
+The file can be created by using generate_mytopo_tpv29.py (https://github.com/SeisSol/Examples/blob/master/tpv29/generate_mytopo_tpv29.py). Save this file as *mytopo\_tpv29*.
```
[the remark this answers is not in the document]
Comment (Thomas-Ulrich) [reply Documentation/tpv29.rst:34]: please add also something like:
"
Here we show a fully opensource workflow which allows generating a mesh accounting for tpv29 rough fault geometry. This yields a mesh that does not properly account for the intersection between fault and the free-surface.  We note that it is here not an important issue, as the tpv29 benchmark does not feature surface rupturing. Another drawback of this workflow is that the rate of mesh size coarsening is not easy parametrizable.
A more straightforward and accurate way to generate a mesh would be to use simModeler. "

File: Documentation/tpv13.rst [hunk 2]
```diff
@@ -3,7 +3,9 @@
 SCEC TPV13
 ==========
 
-TPV13 is similar to TPV12 except for that material properties are **non-associative Drucker-Prager plastic**. TPV13 is using non-associative Drucker-Prager plasticity with yielding in shear. The material is characterized by six constitutive parameters:
+TPV13 is similar to TPV12 except for that material properties are **non-associative Drucker-Prager plastic**. TPV13 is using non-associative Drucker-Prager plasticity with yielding in shear. To run TPV13, it requires to recompile SeisSol with "plasticity = 'yes'" in the configuration python script.
```
Comment (Thomas-Ulrich) [inline Documentation/tpv13.rst:6]: "non-associative Drucker-Prager plastic with yielding in shear".
Remove the 2nd sentence.

File: Documentation/supermuc.rst [hunk 3]
```diff
@@ -229,6 +229,6 @@ set compiler options:
   export ASYNC_BUFFER_ALIGNMENT=8388608
   source /etc/profile.d/modules.sh
 
-  echo $SLURM_NTASKS
-  srun ./SeisSol_release_generatedKernels_dskx_hybrid_none_9_4 parameters.par
+  ech o$SLURM_NTASKS
```
Comment (Thomas-Ulrich) [inline Documentation/supermuc.rst:232]: this is wrong.

File: Documentation/supermuc.rst [hunk 4]
```diff
@@ -213,10 +212,11 @@ set compiler options:
   #SBATCH --nodes=40
   #SBATCH --ntasks-per-node=1
   module load slurm_setup
+  
   #Run the program:
   export MP_SINGLE_THREAD=no
   unset KMP_AFFINITY
-  export OMP_NUM_THREADS=94
+  export OMP_NUM_THREADS=47
```
Comment (Thomas-Ulrich) [inline Documentation/supermuc.rst:219]: hypertheading. do not change.

File: Documentation/supermuc.rst [hunk 5]
```diff
@@ -229,6 +229,6 @@ set compiler options:
   export ASYNC_BUFFER_ALIGNMENT=8388608
   source /etc/profile.d/modules.sh
 
-  echo $SLURM_NTASKS
-  srun ./SeisSol_release_generatedKernels_dskx_hybrid_none_9_4 parameters.par
+  ech o$SLURM_NTASKS
+  srun --export=ALL  ./SeisSol_release_generatedKernels_dskx_hybrid_none_9_4 parameters.par
```
Comment (Thomas-Ulrich) [inline Documentation/supermuc.rst:233]: there is already the line
```
#SBATCH --export=ALL
```

File: Documentation/supermuc.rst [hunk 6]
```diff
@@ -136,34 +136,35 @@ set compiler options:
...
   parallelization             = 'hybrid'
-  #parallelization             = 'mpi'
+  #parallelization            = 'mpi'
   generatedKernels            = 'yes'
-  #measureNodeLevelPerformance = 'none'
   useExecutionEnvironment     = 'yes'
-  order = 4
-  equations='elastic'
-  #equations = 'viscoelastic2'
-  #numberOfMechanisms = 3
```
Comment (Thomas-Ulrich) [inline Documentation/supermuc.rst:149]: leave this comment?

Comment (Thomas-Ulrich) [reply Documentation/supermuc.rst:149]: #numberOfMechanisms = 3

File: Documentation/supermuc.rst [hunk 7]
```diff
@@ -116,17 +116,17 @@ set compiler options:
 
 ::
 
-  $ export FC=mpif90
-  $ export CXX=mpiCC
-  $ export CC=mpicc
+  $ export FC=mpiifort
```
Comment (Thomas-Ulrich) [inline Documentation/supermuc.rst:119]: are you sure you need to change that?

Comment (Thomas-Ulrich) [reply Documentation/supermuc.rst:119]: Anyway, this scons procedure is already obsolete, as we now use cmake (but I know that there is no documentation yet).

**Review by Thomas-Ulrich — CHANGES_REQUESTED** [review]

File: Documentation/supermuc.rst (hunk 7 above)
Comment (Thomas-Ulrich) [reply Documentation/supermuc.rst:119]: Now there is a documentation. I suggest you drop the proposed changes on this file.

**Comment by daisy20170101** [comment]
I think they are all fixed now.

**Comment by Thomas-Ulrich** [comment]
> I think they are all fixed now.

please revert supermuc.rst merge master to your branch and fix the conflicts.

**Comment by daisy20170101** [comment]
I use git merge master but there is nothing to merge.

**Comment by Thomas-Ulrich** [comment]
try :

git fetch

git merge origin/master

File: Documentation/tpv13.rst [hunk 8]
```diff
@@ -3,7 +3,9 @@
 SCEC TPV13
 ==========
 
-TPV13 is similar to TPV12 except for that material properties are **non-associative Drucker-Prager plastic**. TPV13 is using non-associative Drucker-Prager plasticity with yielding in shear. The material is characterized by six constitutive parameters:
+TPV13 is similar to TPV12 except for that material properties are **non-associative Drucker-Prager plastic**. To run TPV13, it requires to recompile SeisSol with "plasticity = 'yes'" in the configuration python script.
```
Comment (Thomas-Ulrich) [inline Documentation/tpv13.rst:6]: SeisSol must be compiled with "-DPLASTICITY=ON".

File: Documentation/tpv12.rst [hunk 9]
```diff
@@ -35,7 +35,7 @@ the square is located 12000 m down-dip (at a depth of 10392.30 m), and
 is centered along-strike.
 
 The geometry is generated with GMSH. All the files that are needed for
-the simulation are provided in
+the simulation are provided in https://github.com/SeisSol/Examples/tree/master/tpv13.
```
Comment (Thomas-Ulrich) [inline Documentation/tpv12.rst:38]: provided at?

File: Documentation/tpv24.rst [hunk 10]
```diff
@@ -37,7 +37,7 @@ side of the main fault. That is, the hypocenter is 8000 m from the
 junction point, and 10 000 m deep.
 
 Figure [fig:tpv24mesh] shows the fault model generated in Gmsh. The mesh
-file can be found at. The mesh can be generated following the detailed
+file can be found at https://github.com/SeisSol/Examples/tree/master/tpv24. The mesh can be generated following the detailed
```
Comment (Thomas-Ulrich) [inline Documentation/tpv24.rst:40]: there is no mesh file there. Maybe remove the sentence?

File: Documentation/tpv29.rst [hunk 11]
```diff
@@ -62,12 +62,25 @@ following general Gmsh process.
...
 ::
 
   & gmsh tpv29_step2.geo -3 -optimize_netgen -o tpv29_step2.msh
   
 option optimize_netgen is necessary for optimizing meshing with good quality.
 
+6.Then convert the .msh file to 3D Gambit neutral file and PUML format as same as shown in TPV5
+
+::
+   $ gmsh2gambit -i tpv5.msh -o tpv5.neu
```
Comment (Thomas-Ulrich) [inline Documentation/tpv29.rst:75]: should not be tpv5 here

File: Documentation/tpv29.rst [hunk 12]
```diff
@@ -62,12 +62,25 @@ following general Gmsh process.
...
+   $ pumgen tpv5.neu tpv5
+
+
+The mesh can be created by using a bash script https://github.com/SeisSol/Examples/blob/master/tpv29/generating_the_mesh.sh.
```
Comment (Thomas-Ulrich) [inline Documentation/tpv29.rst:79]: using the following bash script:

**Review by Thomas-Ulrich — CHANGES_REQUESTED** [review]

File: Documentation/tpv13.rst (hunk 8 above)
Comment (git-taufiq) [reply Documentation/tpv13.rst:6]: .. and the new SeisSol CMake compilation does not use the configuration python script anymore.
Can we just use more general terms? in example: 
To run TPV13, it requires to recompile SeisSol with the plasticity option activated in the configuration file.

**Comment by daisy20170101** [comment]
fixed

**Review by Thomas-Ulrich — APPROVED** [review]

**Pull request merged**